src/App.js

```diff
@@ -1,11 +1,17 @@
 import React from "react";
 import Header from "./components/Header.js";
+import CharacterList from "./components/CharacterList.js";
+import { Route } from "react-router-dom";
+import WelcomePage from "./components/WelcomePage.js";
 
-
-export default function App() {
+function App() {
   return (
     <main>
       <Header />
+      {/* <CharacterList/> */}
+      <Route exact path="/" component={WelcomePage}/>
+      <Route path="/characters" component={CharacterList}/>
```

Comment:
The `<Route/>` components all work properly! We just need some `<Link></Link>` components so the user can go from page to page '' '